src/homework/hw14symbolsAndString/MainGuessTheWord.java

```diff
@@ -0,0 +1,57 @@
+package homework.hw14symbolsAndString;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+import static java.lang.Math.min;
+
+class MainGuessTheWord {
+
+    public static void main(String[] args) {
+        MainGuessTheWord mainGuessTheWord = new MainGuessTheWord();
+        String secretWord = mainGuessTheWord.secretWord(mainGuessTheWord.words);
```

Comment:
Why do you transfer array mainGuessTheWord.words here if it field of class MainGuessTheWord so you can reach the array directly in method secretWord via this.words there`s no need in method aparameter (String[] strings). This point should be fixed as it`s a redundant code. For the momment -5 score for this point.

Reply:
I tried to make method more universal. if we have different arrays with words in class, just to put which one we want. But for our task, I agree, I can simplify